udfs/remote_udfs/nlp/create_bq_function.sh

```diff
@@ -0,0 +1,28 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+PROJECT=$1
+DATASET=$2
+LOCATION=$3
+CONNECTION_NAME=$4
+ENDPOINT=$5
+
+NLP_QUERY=" CREATE OR REPLACE  FUNCTION \`$PROJECT.$DATASET.call_nlp\` (x STRING) RETURNS STRING REMOTE WITH CONNECTION \`$CONNECTION_NAME\` OPTIONS (endpoint = '$ENDPOINT', user_defined_context = [(\"mode\",\"call_nlp\")])"
```

Comment:
Make the UDF name the same as the cloud function entry point (e.g. analyze_sentiment)